software/webdb.yml

```
@@ -0,0 +1,14 @@
+name: WebDB
+website_url: https://webdb.app
+source_code_url: https://gitlab.com/web-db/app
+description: Open Source and Efficient Database IDE.
```

Cleaner description that fits sentence case.
```suggestion
description: Efficient database IDE.
```